dbos/workflows_test.go

```diff
@@ -1085,48 +1110,6 @@ func TestSendRecv(t *testing.T) {
 		if time.Since(start) > 5*time.Second {
 			t.Fatalf("receive workflow took too long to complete, expected < 5s, got %v", time.Since(start))
 		}
-
-		// Send and receive again the same workflows to verify idempotency
-		_, err = sendWf(context.Background(), sendWorkflowInput{
-			DestinationID: receiveHandle.GetWorkflowID(),
-			Topic:         "test-topic",
-		}, WithWorkflowID(handle.GetWorkflowID()))
-		if err != nil {
-			t.Fatalf("failed to send message with same workflow ID: %v", err)
-		}
-		receiveHandle2, err := receiveWf(context.Background(), "test-topic", WithWorkflowID(receiveHandle.GetWorkflowID()))
-		if err != nil {
-			t.Fatalf("failed to start receive workflow with same ID: %v", err)
-		}
-		result, err = receiveHandle2.GetResult(context.Background())
-		if err != nil {
-			t.Fatalf("failed to get result from receive workflow with same ID: %v", err)
-		}
-		if result != "message1-message2-message3" {
-			t.Fatalf("expected received message to be 'message1-message2-message3', got '%s'", result)
-		}
-
-		// Get steps for both workflows and verify we have the expected number
-		sendSteps, err := dbos.systemDB.GetWorkflowSteps(context.Background(), handle.GetWorkflowID())
-		if err != nil {
-			t.Fatalf("failed to get steps for send workflow: %v", err)
-		}
-		receiveSteps, err := dbos.systemDB.GetWorkflowSteps(context.Background(), receiveHandle.GetWorkflowID())
-		if err != nil {
-			t.Fatalf("failed to get steps for receive workflow: %v", err)
-		}
-
-		// Verify the number of steps matches the number of send() and recv() calls
-		// sendWorkflow has 3 Send() calls, receiveWorkflow has 3 Recv() calls
-		expectedSendSteps := 3
-		expectedReceiveSteps := 3
-
-		if len(sendSteps) != expectedSendSteps {
-			t.Fatalf("expected %d steps in send workflow, got %d", expectedSendSteps, len(sendSteps))
-		}
-		if len(receiveSteps) != expectedReceiveSteps {
-			t.Fatalf("expected %d steps in receive workflow, got %d", expectedReceiveSteps, len(receiveSteps))
-		}
```

Comment:
this was testing the workflow idempotency, not the steps -- already tested elsewhere.